### ISSUE TITLE
[BUG] Installation fails due to jupyter_contrib_nbextensions

### ISSUE THREAD

**Issue by bbuzzanga** [aria-tools/ARIA-tools#371]
conda/mamba cannot install `jupyter_contrib_nbextensions` into the ARIA-environment (python 3.9 and 3.10 tested)

The first of a bunch of no `ModuleNotFoundErrors`:
Executing transaction: - Traceback (most recent call last):
  File "/Users/buzzanga/mambaforge/lib/python3.10/site-packages/jupyter_contrib_core/notebook_compat/nbextensions.py", line 6, in <module>
    from notebook.extensions import BaseExtensionApp
ModuleNotFoundError: No module named 'notebook.extensions'
